packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -130,7 +130,7 @@ class Checkbox extends Foundation<ICheckboxHandledProps & ICheckboxManagedClasse
      * Handles onChange as a controlled component
      */
     private handleCheckboxChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
-        if (isUndefined(this.props.checked)) {
+        if (typeof this.props.checked !== "boolean") {
```

Comment:
Can you explain this change?

Reply:
I would say that its one less import the way it's been done here so preferrable

Reply:
The expectation is that "checked" is a boolean. By checking for `undefined`, we're not asserting that our data structure meets our expectations. If `checked` were set to `"foobar"` or `Infinity` or `null`, this could would execute under false assumptions and would do unexpected things.

Reply:
Got it